src/main/java/uk/gov/ch/model/update/trusts/TrustLinkData.java

```diff
@@ -9,11 +9,11 @@
 @Entity
 public class TrustLinkData {
 
-    @Id
     @Column(name = "TRUST_ID")
     @JsonProperty("trustId")
     private Long trustId;
 
+    @Id
```

Comment:
I think a BO can be linked to two trusts so both trustIds and corporateBodyAppointmentIds could appear multiple times. If that is the reason you want to move the @Id and we 100% need an @Id then we may need to find a work around. The there is a table linking bos to trusts and that should have a unique key.

Reply:
Hey yes we've made `ROWNUM` the unique ID and just don't return that